src/cmds/qsub.c

```diff
@@ -4915,6 +4922,16 @@ do_submit(char *retmsg)
 			return (rc);
 	}
 
+	/* 
+	 * get environment variable if -V option is set. Return the code
```

Comment:
Can you repharse it as "get the variable _environ_ which points to an array of pointers to strings of the user environment if -V option is set....".